src/signals/incident/definitions/wizard-step-5-samenvatting.js

```diff
@@ -173,6 +173,7 @@ export default {
       page_summary: {
         meta: {
           value: 'summary',
+          name: 'page_summary',
         },
         render: FormComponents.HiddenInput,
```

Comment:
This line can be removed, right?

Reply:
No it cannot. To make it work, I added the name prop

Reply:
I'm pretty sure the line should be removed; leaving it in will dispatch the `UPDATE_INCIDENT` action and set the `summary` prop in the state with value 'page_summary'. Setting that prop has no effect nor has it any business in the state. As mentioned before, the `page_summary` prop in the `controls` configuration could also be `page_summary: {}`.